Xamarin/SSW.Rewards/SSW.Rewards/ViewModels/ProfileViewModel.cs

```diff
@@ -344,7 +344,7 @@ private async Task AddSocialMediaId(SocialUsernameMessage message)
 
             if (result)
             {
-                MessagingCenter.Send(this, Constants.PointsAwardedMessage);
+                MessagingCenter.Send<object>(this, Constants.PointsAwardedMessage);
```

Comment:
Is there something a little more explicit that you can type this as? `object` isn't very helpful to anyone.

Reply:
Reverted this change